src/components/RichTextEditor/context/RichTextEditorContext.tsx

```diff
@@ -9,12 +9,14 @@ export type RichTextEditorContextProps = {
     styles: Record<string, CSSProperties>;
     position: Position;
     wrapperClassNames: string;
+    editorId: string;
```

Comment:
I don't think the editorId is needed to be stored within the context

Reply:
In PlateWrapper.tsx I need to add the editor id to the wrapper tag. This seems the most straightforward way, unless you have something in mind.